guides/integrations-api/general-concepts/api-versioning.md

```diff
@@ -42,42 +42,42 @@ The Swagger API reference now includes the necessary information about the route
 
 Note the availability information.
 
-Same for fields, here is an example how it would look like:
+Same for fields, here is an example of how it would look like:
 
 ![Availability field](../../../.gitbook/assets/availability_field.png)
 
-### API Expectations
+### API expectations
 
-API expectations can be used as a request header in order to define necessary conditions for the server side. Example conditions could be the Shopware version, the existence of plugins or the version of a plugin. There are some examples:
+API expectations can be used as a request header to define necessary conditions for the server side. Example conditions could be the Shopware version, the existence of plugins, or the version of a plugin. There are some examples:
 
 ```text
 GET /api/test
 sw-expect-packages: shopware/core:~6.4
 ```
 
-This would expect that at least Shopware with the version 6.4 is installed.
+This would expect that atleast Shopware with version 6.4 is installed.
 
 ```text
 GET /api/test
 sw-expect-packages: shopware/core:~6.4,swag/paypal:*
 ```
 
-This would expect that the Shopware version is at least 6.4 and PayPal is installed in any version.
+This would expect that the Shopware version is atleast 6.4 and PayPal is installed in any version.
```

Comment:
Is "atleast" a word?

Reply:
Yes right!
It is used to denote minimum quantity in terms of numbers. But all dictionaries accept it. Hence, replacing it.